changelog/63186.fixed

```diff
@@ -0,0 +1 @@
+Disambiguate str vs tuple containment check
```

Comment:
Can you elaborate what modules this fixes.

Reply:
elaborated, please feel free to ignore, I guess